en_US/deploy/cluster/security.md

```diff
@@ -1,55 +1,58 @@
 # Cluster security
 
-When it comes to the security of the EMQX cluster, there two primary
-aspects to consider.
+Security is the cornerstone of all IoT applications and platforms. EMQX adopts multiple protection mechanisms to ensure data and privacy security, for example, EMQX supports multiple [authentication](../../access-control/authn/authn.md) and [authorization](../../access-control/authz/authz.md) mechanisms on the node level. It also leverages the features of port mapping and TLS encryption to ensure the security of client data transmission, message communication between cluster nodes, and enterprise system integrations.
 
-* Secure the ports each node listens on for clustering.
-* Keep the Erlang cookie secret. See` node.cookie` config.
+This section introduces port mapping in EMQX and how to configure TLS/SSL encryption. 
 
 ::: tip Tip
-It's a good practice to keep the clustering ports internal by configuring
-firewall rules e.g., AWS security groups or iptables.
+It's a good practice to keep the clustering ports internal by configuring firewall rules e.g., [AWS security groups](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html) or [iptables](https://en.wikipedia.org/wiki/Iptables). 
+
+If there is a firewall between the cluster nodes, the conventional listening ports should be allowed for other nodes in the cluster to reach. <!--I think more content should be added about the firewall setting-->
 :::
 
-## Intra-cluster Communication Ports
+## Set Node Cookies
 
-To form a cluster, EMQX nodes need to connect to each other through some conventional port numbers.
+For security concerns, you should change the default cookie settings to `emqxsecretcookie` in `emqx.conf` on all nodes to join the cluster. 
```

Comment:
A secret cookie instead of this cookie

Reply:
not sure how to deal with this comment and the comment below @wivwiv